src/features/hourlies/components/HourlyReadingsDisplay.tsx

```diff
@@ -0,0 +1,151 @@
+          </Typography>
+          <TableContainer>
+            <Table className={classes.table} size="small" aria-label="weather data table">
+              <TableBody>
+                <TableRow>
+                  <TableCell align="left">Date (PDT)</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      <Moment
+                        date={v.datetime}
+                        tz="America/Vancouver"
+                        format="YYYY-MM-DD HH:mm:ss"
+                      />
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Temperature</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.temperature.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Relative Humidity</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.relative_humidity.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Wind Speed</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.wind_speed.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Wind Direction</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.wind_direction.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Barometric Pressure</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.barometric_pressure.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>Precipitation</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.precipitation.toFixed(HOURLY_VALUES_DECIMAL)}
+                    </TableCell>
+                  ))}
+                </TableRow>
+                <TableRow>
+                  <TableCell>FFMC</TableCell>
+                  {values.map(v => (
+                    <TableCell key={v.datetime} align="left">
+                      {v.ffmc ? v.ffmc.toFixed(HOURLY_VALUES_DECIMAL) : null}
```

Comment:
Once you set ffmc optional as I suggest above you can do v.ffmc?.toFixed(HOURLY_VALUES_DECIMAL)